libs/core/src/lib/icon-tab-bar/components/icon-tab-bar-filter-type/icon-tab-bar-filter-type.component.html

```diff
@@ -0,0 +1,60 @@
+<ul
+    class="fd-icon-tab-bar__header"
+    role="tablist"
+    fdOverflowItems
+    [isRtl]="isRtl"
+    itemSelector="[data-overflowItem]"
```

Comment:
Not sure if this selector is good. Maybe we can stick with our directives instead?

Reply:
I would agree, let's do it using directives!